docs/template-provider-router.rb

```diff
@@ -41,10 +41,10 @@
   # Property symbol arrays for method auto-generation. There are separate arrays
   # because the boolean-based methods are processed slightly different.
   X__CONSTANT_NAME__X_NON_BOOL_PROPS = [
-    :X__PROPERTY_INT__X,
+    :X__PROPERTY_INT__X
```

Comment:
Ugh. I thought we had a cop that enforced commas on the last entry? Are we going away from that?